src/v/kafka/server/handlers/produce.h

```diff
@@ -14,12 +14,6 @@
 
 namespace kafka {
 
-struct produce_handler {
-    using api = produce_api;
-    static constexpr api_version min_supported = api_version(0);
-    static constexpr api_version max_supported = api_version(7);
-    static process_result_stages handle(request_context, ss::smp_service_group);
-    static constexpr auto despam_interval = std::chrono::minutes(5);
```

Comment:
despam_interval has moved inside the implementation, there was no reason, I think, to have it here inside the public interface of the handler.